File: lib/models/transferencia_request.dart
class TransferenciaRequest {
  final int idContaRemetente;
  final String numeroContaDestinatario;
  final double valorTransferencia;

  TransferenciaRequest({
    required this.idContaRemetente,
    required this.numeroContaDestinatario,
    required this.valorTransferencia,
  });

  factory TransferenciaRequest.fromJson(Map<String, dynamic> json) {
    return TransferenciaRequest(
      idContaRemetente: json['idContaRemetente'],
      numeroContaDestinatario: json['numeroContaDestinatario'],
      valorTransferencia: json['valorTransferencia'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'idContaRemetente': idContaRemetente,
      'numeroContaDestinatario': numeroContaDestinatario,
      'valorTransferencia': valorTransferencia,
    };
  }
}

File: src/main/java/ifsp/edu/source/Request/TransferenciaRequest.java
package ifsp.edu.source.Request;

public class TransferenciaRequest {
    private long idContaRemetente;
    private String numeroContaDestinatario;
    private double valorTransferencia;

    // Getters e Setters
    public long getIdContaRemetente() {
        return idContaRemetente;
    }

    public void setIdContaRemetente(long idContaRemetente) {
        this.idContaRemetente = idContaRemetente;
    }

    public String getNumeroContaDestinatario() {
        return numeroContaDestinatario;
    }

    public void setNumeroContaDestinatario(String numeroContaDestinatario) {
        this.numeroContaDestinatario = numeroContaDestinatario;
    }

    public double getValorTransferencia() {
        return valorTransferencia;
    }

    public void setValorTransferencia(double valorTransferencia) {
        this.valorTransferencia = valorTransferencia;
    }
}
